README.md

```diff
@@ -1,2 +1,23 @@
 # HW01
-Edit the readme
+Nabilah Sammudin
+7/6/2020
+
+## Brief Biography
+
+My name is **Nabilah** and I am from **Malaysia**. I graduated from **Purdue University** back in 2017 and I am currently working as a research technician in the same lab I did my research as an undergraduate. 
```

Comment:
Hey Nabilah, it's Ariana!  I remember you from recruitment weekend, nice to say hi again!  I hope your research / lab have been doing well, or as well as you can be, during these past few months.